vocabularies/Analytics.json

```diff
@@ -113,14 +131,29 @@
             "$AppliesTo": [
                 "EntityType"
             ],
+            "@Core.Revisions": [
+                {
+                    "Kind": "Deprecated",
+                    "Description": "Deprecated in favor of [`AggregatedProperty`](#AggregatedProperty)"
+                }
+            ],
             "@Core.Description": "Dynamic properties for aggregate expressions with specified aggregation method defined on the annotated entity type.",
             "@Core.LongDescription": "\n            Other annotations may refer in property paths to dynamic properties declared in any AgrgegatedProperties annotation of the \n\t\t\tgiven entity type to leverage the results of the aggregate expression determined in the context of an entity collection of \n\t\t\tthe annotated type.\n          "
         },
+        "AggregatedProperty": {
+            "$Kind": "Term",
+            "$Type": "Analytics.AggregatedPropertyType",
+            "$AppliesTo": [
+                "EntityType"
+            ],
+            "@Core.Description": "Dynamic property for aggregate expression with specified aggregation method defined on the annotated entity type.",
+            "@Core.LongDescription": "A property path to an `AggregatedProperty` annotation can be used to reference the dynamic property holding the aggregated value."
```

Comment:
How should this work? The term declares a new dynamic property with 

~~~js
@Analytics.AggregatedProperty: {
  Name: "foo"
}
~~~

So in JSON data there will be a dynamic property `foo:42` that is not in the model and cannot be referenced via a PropertyPath.

Reply:
A `UI.Chart/Measures` will _not_ reference the dynamic property `foo`, but the annotation path:
```json
"@UI.Chart": {"Measures": ["@Analytics.AggregatedProperty"]}
```

Reply:
Then we have to document in `UI.Chart/Measures` that the property path can either point to 
1. a declared property of primitive type, in which case the run-time values of that property are the measure values, or to
2.  a structured "property" of type `Analytics.AggregatedPropertyType`, in which case the design-time value of that structured "property" points to a dynamic property whose run-time values are the measure values.

To my knowledge clients currently only expect #1, so adding #2 would be a breaking change.

Reply:
That would mean changing the documentation in many many places. I can do that, but I would prefer explaining it at the target annotation. See https://github.com/SAP/odata-vocabularies/blob/2b4eece969576df6ace263fcae47add2f857885a/vocabularies/Analytics.xml#L123-L125